.travis.yml

```diff
@@ -55,9 +55,13 @@ before_cache:
     - brew cleanup
 
 install:
-  - if [[ $TRAVIS_PYTHON_VERSION == 'pypy' ]]; then python -m pip install -U pip; fi
-  - pip --progress-bar ascii install -rtox-requirements.txt
-
+  - |
+    if [[ $TRAVIS_PYTHON_VERSION == 'pypy' ]]; then
+      python -m pip install -U pip
+      pip install -rtox-requirements.txt --progress-bar off
```

Comment:
The progress bar option is needed on the `pip install -U pip` line as well. That's the line failing the build right now.